CustomDeviceAccessor/CPP/README.md

```diff
@@ -0,0 +1,278 @@
+---
+page_type: sample
+urlFragment: custom-device-accessor
+languages:
+  - cpp
+products:
+  - Windows 10
+  - Windows IoT
+  - Windows 10 IoT Enterprise
+description: This sample shows how to access devices from UWP applications using familiar win32 APIs like CreateFile(), ReadFile(), WriteFile(), and DeviceIoControl() with Windows 10 IoT Enterprise.
+---
+
+# Custom Device Accessor
+
+This sample shows how to access devices from UWP applications using familiar win32 APIs like CreateFile(), ReadFile(), WriteFile(), and DeviceIoControl(). Some reasons you may want to do this are:
+
+ - You have existing code that uses CreateFile()
+ - You prefer the programming model of the win32 APIs to Windows.Devices.Custom
+ - Your application needs the performance of the win32 APIs
+ - You need to open the device synchronously, because the device driver requires it, or you need to leverage FastIO
+
+While you will be able to debug and deploy this app on devices you own by sideloading the application, you will not be able to upload an app that uses unsupported APIs to the Windows Store.
+
+The high level steps involved are:
+
+ 1. Create a C++/CX UWP application
+ 2. Set an msbuild property named `_NoWinAPIFamilyApp` to `true` in the vcxproj file
+ 3. Use CreateFile/ReadFile/WriteFile/DeviceIoControl in your code
+ 4. Make the device accessible to appcontainer processes
+
+This walkthrough uses the COM port on MinnowBoardMax as the example device. MinnowBoardMax has 3 COM ports - two high-speed UARTs on the 26-pin header, and a conventional COM port controlled by serial.sys on a separate 6-pin header. The two high speed UARTs are already exposed to UWP applications, but the COM port is not. This walkthrough shows you how to access the COM port from UWP applications.

```

Comment:
Please update references to UP Board from MinnowBoardMax, and make sure hardware description matches UP Board.

Reply:
Updated references for UP Board